cmd/gen/makefile/flag.go

```diff
@@ -0,0 +1,29 @@
+package makefile
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	app = "app"
+)
+
+type flag struct {
+	App    string
+	Output string
+}
+
+func (f *flag) Init(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&f.App, app, "", `Name of the application, this will be the name of the go binary.`)
+	cmd.Flags().StringVar(&f.Output, "output", "", `Where you want the generated Makefile to be saved`)
```

Comment:
That flat I'd skip unless there is a use-case for that. We should generate it as `Makefile` in the current dir.